### PR TITLE
docs(orca): Add MySQL tasks queue example config

### LINKED ISSUE #2012: Orca uses Redis for tasks queue - how to use SQL
The existing docs cover moving almost all of Orca's persistence to SQL: https://spinnaker.io/setup/productionize/persistence/orca-sql/

However the tasks queue still remains on Redis, would be good to show how that can also be moved to SQL.

### PR DESCRIPTION
**Pull request by karlskewes**
Closes #2012 

Per Slack chat: https://spinnakerteam.slack.com/archives/C091CCWRJ/p1598362581058100

Disclaimer that this is untested with Postgres or Cloud SQL

### CODE REVIEW

File: setup/productionize/persistence/orca-sql.md [hunk 1]
```diff
@@ -84,6 +84,27 @@ executionRepository:
...
   activeExecutions:
     redis: false
+
+# Use SQL for Orca's work queue
+# Settings from Netflix and may require adjustment for your environment
+# Only validated with AWS Aurora MySQL 5.7
+# Please PR if you have success with other databases
+keiko:
+  queue:
+    fillExecutorEachCycle: true
```
Comment (karlskewes) [inline setup/productionize/persistence/orca-sql.md]: Could be removed if this is to merge: https://github.com/spinnaker/keiko/pull/151

File: setup/productionize/persistence/orca-sql.md [hunk 2]
```diff
@@ -84,6 +84,27 @@ executionRepository:
...
+    sql:
+      enabled: true
+    redis:
+      enabled: false
+
+queue:
+  zombieCheck:
+    enabled: true
```
Comment (karlskewes) [inline setup/productionize/persistence/orca-sql.md:101]: [Defaults to `false`](https://github.com/spinnaker/orca/blob/master/orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/metrics/ZombieExecutionCheckingAgent.kt#L65) - unsure implications

File: setup/productionize/persistence/orca-sql.md (hunk 1 above)
Comment (robzienert) [reply setup/productionize/persistence/orca-sql.md]: Merged: https://github.com/spinnaker/keiko/releases/tag/v3.8.1

**Review by robzienert — APPROVED** [review]

File: setup/productionize/persistence/orca-sql.md (hunk 1 above)
Comment (karlskewes) [reply setup/productionize/persistence/orca-sql.md]: Thanks, removed it from here then.

**Review by aimeeu — APPROVED** [review]
LGTM

**Pull request merged**